### DISCUSSION TITLE
How to log field QSO (/P or /M) the right way ?

### DISCUSSION THREAD

**Discussion by jpfox** [magicbug/Cloudlog#1551] [Q&A] [answered]
Hi and thank you to devs of CloudLog !

I have created a separated profile for my /P operation. However, I don't know in which field of the QSO form I should input my locator and, eventually, the SOTA/POTA identifier I am (not those of my correspondent).

Have I really to create a Station Location Profile for each location I go ? Can I reuse a profile "Portable operation" without loosing the location for older QSO ?

Thanks for your help.

Jean-Pierre - F1RUM

**Comment by dg9vh** [comment, marked as answer]
Hi,
I think within separat station-locations this would be the right place. I created for every different location with /p-activation a special station location and put the info within this location in the corresponding fields.

Kim
DG9VH

**Reply by jpfox** [reply]
Thanks for your answer. It seems to be complicated with other logging software too... especially when syncing with eqsl, lotw, qrz.com... using several books (one per location) when you often operate in fields is heavy to manage.
Fields describing my station attached to each QSO should be great. ADIF format support these.
Maybe a specific "Field operation" form to log QSO with a  common header for SOTA/location/Power could be a solution.
